db/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/db/PersisterOppgaveLøser.kt

```diff
@@ -1,32 +1,36 @@
 package no.nav.helsearbeidsgiver.inntektsmelding.db
 
 import no.nav.helse.rapids_rivers.JsonMessage
-import no.nav.helse.rapids_rivers.MessageContext
 import no.nav.helse.rapids_rivers.RapidsConnection
 import no.nav.helse.rapids_rivers.River
 import no.nav.helsearbeidsgiver.felles.BehovType
+import no.nav.helsearbeidsgiver.felles.DataFelt
+import no.nav.helsearbeidsgiver.felles.EventName
 import no.nav.helsearbeidsgiver.felles.Key
+import no.nav.helsearbeidsgiver.felles.rapidsrivers.Løser
 
 class PersisterOppgaveLøser(
     rapidsConnection: RapidsConnection,
     val repository: ForespoerselRepository
-) : River.PacketListener {
+) : Løser(rapidsConnection) {
 
-    init {
-        River(rapidsConnection).apply {
-            validate {
-                it.demandAll(Key.BEHOV.str, listOf(BehovType.PERSISTER_OPPGAVE_ID.name))
-                it.requireKey(Key.UUID.str)
```

Comment:
må ikke / bør ikke ha uuid?

Reply:
Det er ikke helt konsistent på tvers av koden hvis UUID er påkrevd eller ikke. I Løser klassen det er definert som it.InterestedId(UUID). Det kan godt vare at UUID er påkrevd men dette er ikke enforset for øyeblikket.